database/migrations/2023_01_07_005110_create_mood_tracking_parameters_table.php

```diff
@@ -29,35 +29,35 @@ public function up()
             foreach ($vaults as $vault) {
                 DB::table('mood_tracking_parameters')->insert([
                     'vault_id' => $vault->id,
-                    'label_translation_key' => trans('vault.settings_mood_tracking_parameters_awesome'),
+                    'label_translation_key' => 'vault.settings_mood_tracking_parameters_awesome',
```

Comment:
If we don't care about the existing vaults in the DB, we can get rid of all these lines since vaults are populated with these values in the `CreateVault` service here 

https://github.com/monicahq/chandler/blob/e03f3b051499172b40553793d2f80992c03c921b/app/Domains/Vault/ManageVault/Services/CreateVault.php#L117

Reply:
It's not wrong indeed